modules/uhttp/client/client_test.go

```diff
@@ -32,10 +33,11 @@ import (
 	"golang.org/x/net/context"
 )
 
-var _defaultUHTTPClient = NewClient(*_defaultHTTPClient)
+var _defaultHTTPClient = &http.Client{Timeout: 2 * time.Second}
```

Comment:
depending on your answer to the other comment you can remove the & here if all you need is a value struct